support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionBuilder.scala

```diff
@@ -70,7 +70,7 @@ object DigitalSubscriptionBuilder {
   )(implicit ec: ExecutionContext): BuildResult = {
 
     val (contractAcceptanceDelay, autoRenew, initialTerm) = if (readerType == Gift)
-      (0, false, purchase.billingPeriod.monthsInPeriod)
+      (0, false, 13) // Expiry for gift redemption codes is 12 months so set the initial term to 13 to allow for this
```

Comment:
will we remember to change this, if later we decide to extend the expiry?
Also wondering if this will affect any reporting or revenue spreading that we do (preumsbaly it will already have to take gifts into account specially)

Reply:
I've updated this so that it is explicitly dependent on the expiry time so that if we change that then this will update automatically.
Yes reporting will have to be updated anyway.

Reply:
👍